src/main/kotlin/lotto/domain/Money.kt

```diff
@@ -0,0 +1,18 @@
+package lotto.domain
+
+class Money(money: Long) {
+    val value: Long = validateMoney(money)
+
+    private fun validateMoney(money: Long): Long {
+        require(money >= MINIMUM_MONEY)
+        return money
+    }
```

Comment:
```suggestion
class Money(private val money: Long) {
    init {
        validateMoney(money)
    }
```

init method를 사용하면 더 깔끔해지지 않을까요?